op-service/txmgr/txmgr.go

```diff
@@ -196,9 +262,15 @@ func (m *SimpleTxManager) Send(ctx context.Context, updateGasPrice UpdateGasPric
 				continue
```

Comment:
Don't we want to bump the gas price if `IsWaitingForConfirmation`? This feels like we want to wait until a threshold passes then increase the gas price if still above basefee or always increase if below the basefee.

Or we want an interface to allow for different replacement strategies. Prob over optimizing for now